crates/trigger/src/cli.rs

```diff
@@ -147,29 +156,44 @@ where
         loader: impl Loader + Send + Sync + 'static,
         locked_url: String,
     ) -> Result<Executor> {
-        let trigger_config =
-            TriggerExecutorBuilderConfig::load_from_file(self.runtime_config_file.clone())?;
+        let runtime_config = self.build_runtime_config()?;
 
         let _sloth_warning = warn_if_wasm_build_slothful();
 
         let mut builder = TriggerExecutorBuilder::new(loader);
         self.update_wasmtime_config(builder.wasmtime_config_mut())?;
 
-        let logging_hooks = StdioLoggingTriggerHooks::new(self.follow_components(), self.log_dir());
+        if runtime_config.state_dir().is_none() {
+            println!("No local state dir; logs and key value data may not be written to disk.");
+            println!("You can specify a state dir with `spin-up --state-dir <dir>`.\n");
+        }
+
+        let log_dir = runtime_config.log_dir();
+        if let Some(log_dir) = &log_dir {
+            println!("Logging component stdio to {:?}\n", log_dir.join(""))
```

Comment:
Is this only component stdio?  Spin's own logs are... transient? or written somewhere else?

Reply:
We've never written Spin's own logs anywhere. I considered adding `<log-dir>/spin.log` but this PR is already an overgrown mess.